new-filename.js

```diff
@@ -0,0 +1,3 @@
+if(foo === "bar") {
+  console.log('something new for real2')
```

Comment:
Codacy has a fix for the issue: [Replace `'something·new·for·real2')` with `"something·new·for·real2");`](https://app.dev.codacy.org/gh/troubleshoot-codacy/no-suggestions/pullRequest?prid=30887)
```suggestion
  console.log("something new for real2");
```